Update Lecture 4

EvZissel opened the pull request:
1. line 2: involves -> involve, lecture -> chapter
2. line 19 (4.1 Markov Chains: Reminder): sum over i for all j -> sum of j for all i. since the each row sums to 1 in stochastic matrices
3. lines 22-23 (finite dimensional distribution): initial state i -> i₀
4. line 33 (State classification): missing brackets for integer 
5. line 56 (Ergodicity): notation consistency π(i) -> πᵢ
6. line 73 (Reversible Markov chains): changed i,j to ∀i,j
7. line 91 (Discrete-time-queue): changed i to ∀i
8. line 166: missing comma 
9. line 231 (Finite Horizon Problems - expected return value): resized brackets
10. line 232: maximized -> maximizes (tense)
11. line 236: added colon (consistency)
12. line 245 (time horizon): added missing brackets
13. line 250 (Discounted return): resized brackets
14. line 252 (Average return): to maximize -> in maximizing (tense)
15. line 253: resize brackets
16. line 262: resize brackets
17. line 299 (state probability): added * for optimal Markov policy
18. line 310 (Value Iteration): added brackets
19. line 399 (The Q function): fix time index k -> k + 1

EvZissel committed:
Update Lecture 4

1. line 2: involves -> involve, lecture -> chapter
2. line 19 (4.1 Markov Chains: Reminder): sum over i for all j -> sum of j for all i. since the each row sums to 1 in stochastic matrices
3. lines 22-23 (finite dimensional distribution): initial state i -> i₀
4. line 33 (State classification): missing brackets for integer 
5. line 56 (Ergodicity): notation consistency π(i) -> πᵢ
6. line 73 (Reversible Markov chains): changed i,j to ∀i,j
7. line 91 (Discrete-time-queue): changed i to ∀i
8. line 166: missing comma 
9. line 231 (Finite Horizon Problems - expected return value): resized brackets
10. line 232: maximized -> maximizes (tense)
11. line 236: added colon (consistency)
12. line 245 (time horizon): added missing brackets
13. line 250 (Discounted return): resized brackets
14. line 252 (Average return): to maximize -> in maximizing (tense)
15. line 253: resize brackets
16. line 262: resize brackets
17. line 299 (state probability): added * for optimal Markov policy
18. line 310 (Value Iteration): added brackets
19. line 399 (The Q function): fix time index k -> k + 1

avivt commented on Lecture4.tex:
```diff
@@ -16,11 +16,11 @@ \section{Markov Chains: A Reminder}
 
-Given the initial distribution ${p_0}$ of ${X_0}$, namely $p({X_0} = i) = {p_0}(i)$, we obtain the finite-dimensional distributions:
-\[{\cal P}({X_0} = {i_0}, \ldots ,{X_t} = {i_t}) = {p_0}(i){p_{{i_0}{i_1}}} \cdot  \ldots  \cdot {p_{{i_{t - 1}}{i_t}}}.\]
+Given the initial distribution ${p_0}$ of ${X_0}$, namely $p({X_0} = i_0) = {p_0}(i_0)$, we obtain the finite-dimensional distributions:
```
i is OK here (general element in the vector). Only in the next line it should be i_0

avivt commented on Lecture4.tex:
```diff
@@ -30,7 +30,7 @@ \section{Markov Chains: A Reminder}
 \item A communicating class (or just class) is a maximal collection of states that communicate.
 \item The Markov chain is irreducible if all states belong to a single class (i.e., all states communicate with each other).
-\item State $i$ is periodic with period $d \ge 2$ if  $p_{ii}^{(m)} = 0$ for $m \ne \ell d$ and  $p_{ii}^{(m)} > 0$ for $m = \ell d$, $\ell  = 1,2, \ldots $.   Periodicity is a class property: all states in the same class have the same period.
+\item State $i$ is periodic with period $d \ge 2$ if  $p_{ii}^{(m)} = 0$ for $m \ne \ell d$ and  $p_{ii}^{(m)} > 0$ for $m = \ell d$, $\ell = \{ 1,2, \ldots \}$. Periodicity is a class property: all states in the same class have the same period.
```
if written as a set, then change = to \in

avivt commented on Lecture4.tex:
```diff
@@ -242,15 +242,15 @@ \subsection{Finite Horizon Problems}
 
 \subsection{Infinite Horizon Problems}
-We next consider planning problems that extend to an unlimited time horizon, $t = 0,1,2, \ldots $. Such planning problems arise when the system in question is expected to operate for a long time, or a large number of steps, possibly with no specific "closing" time.
+We next consider planning problems that extend to an unlimited time horizon, $t = \{0,1,2, \ldots \}$. Such planning problems arise when the system in question is expected to operate for a long time, or a large number of steps, possibly with no specific "closing" time.
```
this is not required

avivt commented on Lecture4.tex:
```diff
@@ -307,7 +307,7 @@ \subsection{Dynamic Programming for Policy Evaluation}\label{sss:pol_eval}
 \begin{lemma}[\textbf{Value Iteration}]\label{lem:finite_horizon_VI} $V_k^\pi (s)$ may be computed by the backward recursion:
 \[V_k^\pi (s) = {\left\{ {{r_k}(s,a) + \sum\nolimits_{s' \in {S_{k + 1}}} {{p_k}(s'|s,a)} \;V_{k + 1}^\pi (s')} \right\}_{a = {\pi _k}(s)}}\;,\quad s \in {S_k}\]
-for $k = T - 1, \ldots ,0$,  starting with  $V_T^\pi (s) = {r_T}(s)$.
+for $k = \{ T - 1, \ldots ,0 \}$,  starting with  $V_T^\pi (s) = {r_T}(s)$.
```
don't need this as a set

EvZissel committed:
Fix commit for Update Lecture 4

3. lines 22-23: revert change - initial state i -> i₀
4. line 33 : changed from '\ell =' to '\ell  \in'
12. line 245:  revert change - added missing brackets
18. line 310: revert change - added brackets

EvZissel commented:
I incorporated your feedback regarding items:
3. lines 22-23: revert change - initial state i -> i₀
4. line 33 : changed from '\ell =' to '\ell  \in'
12. line 245:  revert change - added missing brackets
18. line 310: revert change - added brackets

avivt commented:
Thanks you!
